### PR TITLE
Some tweaks for dp-terraform charts

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: dp-terraform/helm/rhacs-terraform/values.yaml [hunk 1]
```diff
@@ -113,6 +113,7 @@ observability:
     key: ""
   deadMansSwitch:
     url: ""
+  resyncPeriod: 10m
```
Comment (ludydoo) [inline dp-terraform/helm/rhacs-terraform/values.yaml:116]: @stehessel for diminishing the resync period ^

File: dp-terraform/helm/rhacs-terraform/values.yaml [hunk 2]
```diff
@@ -246,12 +247,24 @@ external-secrets:
     image:
       repository: quay.io/app-sre/external-secrets
       tag: v0.9.5
+    tolerations:
+      - key: node-role.kubernetes.io/acscs-infra
+        operator: Exists
+        effect: NoSchedule
+    nodeSelector:
+      node-role.kubernetes.io/acscs-infra: ""
```
Comment (ludydoo) [inline dp-terraform/helm/rhacs-terraform/values.yaml:255]: This is an additional workload we want to schedule on acscs-infra nodes

File: dp-terraform/helm/rhacs-terraform/values.yaml [hunk 3]
```diff
@@ -246,12 +247,24 @@ external-secrets:
...
       runAsUser: null
     image:
       repository: quay.io/app-sre/external-secrets
       tag: v0.9.5
+    tolerations:
+      - key: node-role.kubernetes.io/acscs-infra
+        operator: Exists
+        effect: NoSchedule
+    nodeSelector:
+      node-role.kubernetes.io/acscs-infra: ""
```
Comment (ludydoo) [inline dp-terraform/helm/rhacs-terraform/values.yaml:267]: This is an additional workload we want to schedule on acscs-infra nodes

File: dp-terraform/helm/rhacs-terraform/charts/logging/templates/02-eventrouter-03-deployment.yaml [hunk 4]
```diff
@@ -23,14 +23,22 @@ spec:
...
       serviceAccount: eventrouter
+      {{- if .Values.affinity }}
+      affinity: {{- toYaml .Values.affinity | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- toYaml .Values.tolerations | nindent 8 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
+      {{- end }}
```
Comment (ludydoo) [inline dp-terraform/helm/rhacs-terraform/charts/logging/templates/02-eventrouter-03-deployment.yaml:34]: This is an additional workload we want to schedule on acscs-infra nodes

File: dp-terraform/helm/rhacs-terraform/values.yaml [hunk 5]
```diff
@@ -246,12 +247,24 @@ external-secrets:
     image:
```
Comment (kovayur) [inline dp-terraform/helm/rhacs-terraform/values.yaml:247]: Shouldn't we delete `logging.tolerations` and `logging.nodeSelector` above (lines 126-131)?

Comment (ludydoo) [reply dp-terraform/helm/rhacs-terraform/values.yaml:248]: nice find

**Review by stehessel — APPROVED** [review]

**Review by kovayur — APPROVED** [review]

**Comment by ebensh** [comment]
Do we want to revive & merge? Close out?

**Pull request closed without merging**